test/task_serialize_test.py

```diff
@@ -0,0 +1,113 @@
+"""
+We want to test that task_id is consistent when generated form:
+
+ 1. A real task instance
+ 2. The task_family and a dictionary of parameter values (as strings)
+ 3. A json representation of #2
+
+We use the hypothesis package to do property-based tests.
```

Comment:
Cool! :D :D 

I love the functional programming influence luigi is getting more and more :)